pandas/core/generic.py

```diff
@@ -4283,8 +4283,16 @@ def _align_series(self, other, join='outer', axis=None, level=None,
 
         Parameters
         ----------
-        cond : boolean %(klass)s or array
-        other : scalar or %(klass)s
+        cond : boolean %(klass)s, array or callable
+            If cond is callable, it is computed on the %(klass)s and
+            should return boolean %(klass)s or array.
+            The callable must not change input %(klass)s
```

Comment:
Do you know a format to use ``versionadded`` as a part of an argument description? Adding it under ``cond`` looks ``cond`` argument itself is added in the specified version.